jenkins/automation/kubernetes/upgrade-cortx-cluster.groovy

```diff
@@ -0,0 +1,196 @@
+pipeline {
+    agent {
+        node {
+            label 'docker-k8-deployment-node'
+        }
+    }
+    
+    options {
+        timeout(time: 240, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '30', numToKeepStr: '30'))
+        ansiColor('xterm')
+    }
+
+    parameters {
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch or GitHash for Cluster Setup scripts', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repository for Cluster Setup scripts', trim: true)
+        string(name: 'CORTX_SERVER_IMAGE', defaultValue: 'ghcr.io/seagate/cortx-rgw:2.0.0-latest', description: 'CORTX-SERVER image', trim: true)
```

Comment:
Please update the description to `CORTX-SERVER image to be updated`. Please update all image input parameters.